style.css

```diff
@@ -24,3 +24,12 @@ img {
 h1 {
   color: #ffe3dc;
 }
+
+.hello {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 100px;
```

Comment:
Please don't use `px` as unit for `font-size`, since it is very static and won't adjust to user preference. Here's an article about this: https://blog.logrocket.com/the-elements-of-responsive-typography/